### DISCUSSION TITLE
posts/my-car/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#55] [Comments]
# posts/my-car/

俺拥有属于自己的小车啦！
[&hellip;] 对物的期待，本质是对一种生活的向往。就车而言，它是移动的空间，可以让我们不受风吹雨打，可以说走就走。俺和小王从去年就已经开始盘算，也试驾过两回：
[&hellip;] 但是，俺的小车并不是它们之一。关于它的来历，得容我好好捋一捋：
[&hellip;] 于是乎，上周姐姐购入了 Model Y，把小白过户给俺，俺沿用「小白」这个名字。估价为 6.5w， &hellip;

https://shitao5.org/posts/my-car/

**Comment by hongtaoh** [comment]
我的小绿是不到2.5 万人民币买的。和我的比你这已经是豪车了。

**Reply by Shitao5** [reply]
不愧是写二手车经验贴的人，我要发出和王云子一样的[感叹](https://github.com/hongtaoh/hongtaoh.github.io/discussions/168#discussioncomment-9984567)了。自己满意的就是最好的，已经想象到小绿载着鸿涛奔走在美国大地上的景象了哈哈。
